corehq/motech/repeaters/tests/test_models.py

```diff
@@ -392,6 +395,75 @@ def test_cached_attempts(self):
             self.assertEqual(len(self.repeat_record.attempts), 2)
 
 
+class TestRepeaterHandleResponse(RepeaterTestCase):
+
+    @contextmanager
+    def repeat_record(self):
+        repeat_record = self.repeater.repeat_records.create(
+            domain=DOMAIN,
+            payload_id='eggs',
+            registered_at=timezone.now(),
+        )
+        try:
+            yield repeat_record
+        finally:
+            repeat_record.delete()
```

Comment:
We don't actually have to handle cleanup for a SQL Table in a `TestCase` class since all SQL updates happen within a transaction that is rolled back at the end of the test right? Could this be simplified to not use a context manager or is there a benefit to this I'm missing?

Reply:
It could totally just create the repeat record. I'll change it.

Reply:
https://github.com/dimagi/commcare-hq/pull/35165/commits/aeb3277e62e5c01399aa051cb66ed00c271c67fa